### ISSUE TITLE
[BUG] [SP-API] [Issue] searchcatalogitems return irrelevant items

### ISSUE THREAD

**Issue by ganeshkrishnan1** [amzn/selling-partner-api-docs#3259]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X ] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

searchcatalogitems should return the catalog as per amazon.com search results. For eg searching for "boy mom candle" should return relevant products which amazon.com correctly finds. But this api returns catalog items which are not in any of the search results while searching  on amazon.com
I tried with url encoding the keyword to see if there was an encoding issue but it's the same problem.

tldr: The results of the searchcatalogitems do not match amazon search results. In fact there is no intersection and this api returns random items that match this keyword.

## SP-API Name and Version or Report/Feed Type
https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-reference#searchcatalogitems
## Expected Behavior

calling this api with a keyword should match amazon search results.
eg calling sp-api for NA region with keyword "boy mom candle" should return the catalog items that match amazon.com search results.

## Current Behavior

Currently it returns catalog items that are very low match for the keywords. The returned catalog items are nowhere in the search results on customer facing amazon
